backend/app/tests/crud/collections/collection/test_crud_collection_read_one.py

```
@@ -8,15 +8,15 @@
 from app.models import Collection
 from app.tests.utils.document import DocumentStore
 from app.tests.utils.utils import get_project
-from app.tests.utils.collection import get_collection
+from app.tests.utils.collection import get_assistant_collection
 
 
 def mk_collection(db: Session) -> Collection:
     openai_mock = OpenAIMock()
     project = get_project(db)
     with openai_mock.router:
         client = OpenAI(api_key="sk-test-key")
```

Ruff (0.14.13)

Local variable `client` is assigned to but never used

Remove assignment to unused variable `client`

(F841)